templates/solutions/secure-open-source/index.html

```diff
@@ -641,20 +641,18 @@ <h2>For enterprise clouds with secure open source foundations</h2>
         <div class="col-3 col-medium-2">
           <h3 class="p-heading--5">
             {{ image (
-              url="https://assets.ubuntu.com/v1/635eea3b-Canonical Product Logo kubernetes.png",
+              url="https://assets.ubuntu.com/v1/c71f8d79-kubernetes-logo.png",
               alt="Canonical Kubernetes",
-              width="810",
-              height="121",
+              width="151",
+              height="48",
               hi_def=True,
-              loading="lazy",
-              attrs={"style": "max-width: 250px;"}
+              loading="lazy"
               ) | safe
             }}
           </h3>
         </div>
         <div class="col-6 col-medium-3">
-          <p>Benefit from the latest upstream features with the security guarantees you need. Use features like strict confinement to ensure that vulnerabilities remain isolated to a single application, with seamless remote resolution.</p>
-          <p>Landscape reduces the cost of managing open source:</p>
+          <p>Benefit from the latest upstream features with the security guarantees you need. Use features like strict confinement to ensure that vulnerabilities remain isolated to a single application, with seamless remote resolution. Landscape reduces the cost of managing open source:</p>
```

Comment:
I think "Landscape reduces the cost of managing open source" needs to go
```suggestion
          <p>Benefit from the latest upstream features with the security guarantees you need. Use features like strict confinement to ensure that vulnerabilities remain isolated to a single application, with seamless remote resolution.</p>
```